src/api/flash/TeensyFlasher.js

```diff
@@ -20,13 +20,23 @@ import TeensyLoader from "teensy-loader";
 
 import { delay, toStep } from "./utils";
 
-export const TeensyFlasher = async (board, port, filename, options) => {
+const rebootToNormal = async (port) => {
+  logger("flash").debug("rebooting to normal mode");
+  const device = await TeensyLoader.open(0x16c0, 0x0478);
```

Comment:
what are those constants? let's name em.

Reply:
that's the HalfKay bootloader vendor/prod ids, and yes we can name them.